test/server/pglib.py

```diff
@@ -21,7 +21,7 @@
 # into a cookie jar.
 def attach_device(device_id, account_id):
     dbname = pg_vars["PGDATABASE"] or pg_vars["PGUSER"]
-    conn = psycopg2.connect(dbname=dbname, user=pg_vars["PGUSER"], host=pg_vars["PGHOST"], port=pg_vars["PGPORT"])
+    conn = psycopg2.connect(dbname=dbname, user=pg_vars["PGUSER"], host=pg_vars["PGHOST"], port=pg_vars["PGPORT"], password="postgres")
```

Comment:
I'm guessing this is a local configuration issue, but it shouldn't be necessary. It might break tests for other installations, though I'm not sure.

Reply:
that's correct, updated PR to remove explicitly adding password